styles/globals.css

```diff
@@ -0,0 +1,39 @@
+@import url('https://fonts.googleapis.com/css?family=Dosis:400,600,700');
+
+:root{
```

Comment:
move this to color.js file and define styled-components theme to use these in styles.